soccer/WorldState.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <Constants.hpp>
+#include <Geometry2d/Pose.hpp>
+
+/**
+ * @brief Contains robot motion state data
+ * @details This class contains data that comes from the vision system
+ * including position data and which camera this robot was seen by and
+ * what time it was last seen.
+ */
+struct RobotState {
+    Geometry2d::Pose pose;
+    Geometry2d::Twist velocity;
+    RJ::Time timestamp;
+    bool visible = false;
+    bool velocity_valid = false;
```

Comment:
I'm 99% this is never actually used anywhere, it's up to you if you want to just remove it

Reply:
It's currently not used anywhere but I'd rather publish it anyway in the chance that we need it eventually.